include/RAJA/util/TypedViewBase.hpp

```diff
@@ -101,15 +106,20 @@ namespace internal
   template<typename ... ARGS>
   struct count_num_tensor_args{
     static constexpr camp::idx_t value =
+#if defined(RAJA_ENABLE_EXPTVECTOR)
         RAJA::sum<camp::idx_t>(
             (internal::expt::isTensorIndex<ARGS>() ? 1 : 0) ...);
+#else
+        0;  // There should be 0 Tensor indices if not vectorizing.
+#endif
   };
 
+#if defined(RAJA_ENABLE_EXPTVECTOR)
   /*
    * Returns which argument has a vector index
    */
   template<camp::idx_t DIM, typename ... ARGS>
-  struct GetTesorArgIdx{
+  struct GetTensorArgIdx{
```

Comment:
Good catch here and throughout. We should run the code through a spell checker for one of the CI checks 😄